### PR TITLE
Updated rtd url from org to io

### PR DESCRIPTION
**Pull request by dpshelio**
readthedocs has sent an email on changing their urls to a different
domain. This is the update to all the instances.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ Next, use git to grab the latest version of SunPy:
 
 Done!
 
-For detailed installation instructions, see the [installation guide](http://sunpy.readthedocs.org/en/latest/guide/installation/index.html)
+For detailed installation instructions, see the [installation guide](http://sunpy.readthedocs.io/en/latest/guide/installation/index.html)
```
Comment (Cadair) [inline README.md]: This should be `docs.sunpy.org`

File: README.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ Getting Help
 
 For more information or to ask questions about SunPy, check out:
 
- * [SunPy Documentation](http://sunpy.readthedocs.org/en/latest/)
+ * [SunPy Documentation](http://sunpy.readthedocs.io/en/latest/)
```
Comment (Cadair) [inline README.md]: this should be `docs.sunpy.org`

File: README.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ Contributing
 
 If you would like to get involved, start by joining the
 [SunPy mailing list](https://groups.google.com/forum/#!forum/sunpy)
-and check out the [Developer's Guide](http://sunpy.readthedocs.org/en/latest/dev.html) section
+and check out the [Developer's Guide](http://sunpy.readthedocs.io/en/latest/dev.html) section
```
Comment (Cadair) [inline README.md]: this should be `docs.sunpy.org`

File: doc/source/dev.rst [hunk 4]
```diff
@@ -344,7 +344,7 @@ Coding Standards
 All code that is part of the SunPy project should follow The Style Guide for
 Python (`PEP 8 <http://www.python.org/dev/peps/pep-0008/>`_) and
 the `coding style and conventions proposed by Astropy
-<https://astropy.readthedocs.org/en/stable/development/codeguide.html#coding-style-conventions>`_.
+<https://astropy.readthedocs.io/en/stable/development/codeguide.html#coding-style-conventions>`_.
```
Comment (Cadair) [inline doc/source/dev.rst]: this should be `docs.astropy.org`

File: doc/source/guide/acquiring_data/database.rst [hunk 5]
```diff
@@ -720,7 +720,7 @@ When using the :class:`sunpy.net.vso.attrs.Wave` attribute, you have to
 specify a unit using ``astropy.units.Quantity``. If not an error is
 raised. This also implies that there is no default unit that is
 used by the class. To know how you can specify a detail using astropy
-check `astropy.units <https://astropy.readthedocs.org/en/stable/units/index.html>`_.
+check `astropy.units <https://astropy.readthedocs.io/en/stable/units/index.html>`_.
```
Comment (Cadair) [inline doc/source/guide/acquiring_data/database.rst]: this should be an intersphinx link i.e. 
```
`astropy.units`
```

File: doc/source/guide/acquiring_data/vso.rst [hunk 6]
```diff
@@ -85,7 +85,7 @@ similarly wavemin(for minimum value) for the query. Also the ``u.AA``
 part comes from ``astropy.units.Quantity`` where `AA` is Angstrom. It
 should be noted that specifying spectral units in arguments is
 necessary or an error will be raised. To know more check
-`astropy.units <https://astropy.readthedocs.org/en/stable/units/index.html>`_.
+`astropy.units <https://astropy.readthedocs.io/en/stable/units/index.html>`_.
```
Comment (Cadair) [inline doc/source/guide/acquiring_data/vso.rst]: Should be intersphinx

File: doc/source/guide/data_types/maps.rst [hunk 7]
```diff
@@ -97,7 +97,7 @@ To get a list of all of the attributes check the documentation by typing::
     >>> help(my_map)   # doctest: +SKIP
 
 From SunPy version 0.6 many attributes return `~astropy.units.quantity.Quantity`
-objects, please refer to the `Astropy documentation <http://astropy.readthedocs.org/en/latest/units/>`_.
+objects, please refer to the `Astropy documentation <http://astropy.readthedocs.io/en/latest/units/>`_.
```
Comment (Cadair) [inline doc/source/guide/data_types/maps.rst]: should be intersphinx

**Comment by dpshelio** [comment]
done! :)

**Comment by Cadair** [comment]
:+1:

**Pull request merged**